src/store/auth/index.ts

```diff
@@ -0,0 +1,4 @@
+import { reducer as authReducer } from './slice';
+import { signIn } from './actions';
```

Comment:
authActions already include signIn
```suggestion
import { reducer as authReducer, actions as authActions } from './slice';
```